react-native/README.md

```diff
@@ -0,0 +1,47 @@
+# Quick Base Craft Demo | React Native
+
+## Craft Demo Presentation Outline
+
+We have created a PowerPoint template which covers an outline of what we would like to talk about during the first section of your interview before you share your craft demo. Feel free to use the template provided or make your own. We are more interested in your code than your PowerPoint skills, so plan your prep time accordingly.
+
+[PowerPoint Template and Craft Demo Outline](https://github.com/QuickBase/interview-demos/blob/master/QuickBase_CraftDemo_PresentationTemplate.pptx)
+
+One of our Quick Base partners has asked us to create a feature inside our react-native application. 
+After an internal review, we have found a significant portion of our user base that could benefit from their feature. 
+You have been given the opportunity to create a proof of concept. 
+
+## General Description
+
+The purpose of the craft demo is to give us a jumping-off-point for discussion of your approach to solving a problem and of your code. We are interested in learning how you approach this problem and go about solving it.
+
+We expect this project to take experienced developers about two to four hours to complete. We know not everyone has unlimited time to work on interview projects and we do not expect that everyone completes all of the requirements. However, you may be asked how you might have solved a requirement you missed had you been able to work on this project without time constraints. Additionally, if you want to extend the project beyond the listed requirements (e.g., tracking additional data points in the 'Tweet Tracker' app), please feel free to do so as you see fit. Please note that we prefer higher-code quality over additional features if you start to run out of time.
```

Comment:
Copy/pasta error: ~(e.g., tracking additional data points in the 'Tweet Tracker' app)~ (e.g., completing some of the stretch goals or creating your own requirements)

Reply:
mmm pasta